src/components/script/ScriptEventBlock.js

```diff
@@ -324,69 +326,71 @@ class ScriptEventField extends Component {
   render() {
     const { eventId, field, value, args } = this.props;
 
-    const inputField = field.multiple ? (
-      value.map((_, valueIndex) => {
-        const fieldId = genKey(eventId, field.key, valueIndex);
-        return (
-          <span key={fieldId} className="ScriptEventBlock__InputRow">
-            <ScriptEventInput
-              id={fieldId}
-              type={field.type}
-              field={field}
-              index={valueIndex}
-              value={value[valueIndex]}
-              args={args}
-              onChange={this.onChange}
-            />
-            <div className="ScriptEventBlock__BtnRow">
-              {valueIndex !== 0 && (
-                <div
-                  className="ScriptEventBlock__Btn"
-                  onClick={this.onRemoveValue(valueIndex)}
-                >
-                  -
-                </div>
-              )}
-              <div
-                className="ScriptEventBlock__Btn"
-                onClick={this.onAddValue(valueIndex)}
-              >
-                +
-              </div>
-            </div>
-          </span>
-        );
-      })
-    ) : (
-      <ScriptEventInput
-        id={genKey(eventId, field.key)}
-        type={field.type}
-        field={field}
-        value={value}
-        args={args}
-        onChange={this.onChange}
-      />
-    );
+    let label = field.label;
+    if (label && label.replace) {
+      label = label.replace(/\$\$([^$]*)\$\$/g, (match, key) => args[key] || 0);
+    }
 
-    if (field.toggleLabel) {
```

Comment:
I think by removing this condition the Avatar field is always visible in Text Dialogue events.

Reply:
Ah, oops. This was quite a tricky merge, I knew I'd miss something! Fixed in f7a8232ba7c9f2a4bedfb80a8744b9adec708aad thanks